Sources/Views/InputTextView.swift

```diff
@@ -38,47 +60,51 @@ open class InputTextView: UITextView {
         return label
     }()
     
-    open override var text: String! {
-        didSet {
-            placeholderLabel.isHidden = !text.isEmpty
-        }
-    }
-
-    private var placeholderLabelConstraintSet: NSLayoutConstraintSet?
-
+    /// The placeholder text that appears when there is no text
     open var placeholder: String? = "New Message" {
         didSet {
             placeholderLabel.text = placeholder
         }
     }
-
+    
+    /// The placeholderLabel's textColor
     open var placeholderTextColor: UIColor? = .lightGray {
         didSet {
             placeholderLabel.textColor = placeholderTextColor
         }
     }
-
+    
+    /// The UIEdgeInsets the placeholderLabel has within the InputTextView
+    open var placeholderLabelInsets: UIEdgeInsets = UIEdgeInsets(top: 4, left: 7, bottom: 4, right: 7) {
+        didSet {
+            updateConstraintsForPlaceholderLabel()
+        }
+    }
+    
+    /// The font of the InputTextView. When set the placeholderLabel's font is also updated
     open override var font: UIFont! {
         didSet {
             placeholderLabel.font = font
         }
     }
-
+    
+    /// The textAlignment of the InputTextView. When set the placeholderLabel's textAlignment is also updated
     open override var textAlignment: NSTextAlignment {
         didSet {
             placeholderLabel.textAlignment = textAlignment
         }
     }
-
-    open var placeholderLabelInsets: UIEdgeInsets = UIEdgeInsets(top: 4, left: 7, bottom: 4, right: 7) {
+    
+    /// The textContainerInset of the InputTextView. When set the placeholderLabelInsets is also updated
+    open override var textContainerInset: UIEdgeInsets {
```

Comment:
This one kind of confuses me. Why even have two properties if we're going to keep them in sync? `textContainerInset` will always override what you've done in `placeholderLabelInsets`. We should keep these separate?